targets/targets.json

```diff
@@ -8088,5 +8088,29 @@
         "detect_code": ["1703"],
         "macros_add": ["GD32E10X"],
         "release_versions": ["5"]
+    },
+    "BLUENRG2": {
+        "inherits": ["Target"],
+        "core": "Cortex-M0",
+        "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
+        "public": false,
+        "extra_labels": ["STMBLUE","CORDIO"],
+        "macros": ["BLUENRG2_DEVICE"],
+        "features": ["BLE"]        
+    },
+    "STEVAL_IDB008V2": {
+        "inherits": ["BLUENRG2"],
+        "extra_labels_add": ["BLUENRG2"],
+        "release_versions": ["5"],
```

Comment:
There's mention about size limitation (withour RTOS ), but adding here version 5? Can you elaborate please

Reply:
@cmonr thank you, it seems that I've violated something about the hierarchy in the target.json file. I'll fix and update on my next commit.

@0xc0170 about the size limitation, it is not feasible to use the Mbed OS 5 RTOS feature, but from what I've understood, since version 5.6 event queue feature (and thus code examples based on it) is available in non-RTOS solutions too.

Together with @avilei and @donatieng decided to develop the non-RTOS solution on Mbed 5.